pom.xml

```diff
@@ -66,7 +67,7 @@
         <surefire.plugin.version>2.22.1</surefire.plugin.version>
         <failsafe.version>2.22.1</failsafe.version>
         <checkstyle.plugin.version>3.1.0</checkstyle.plugin.version>
-        <checkstyle.phase>verify</checkstyle.phase>
+        <checkstyle.phase>prepare-package</checkstyle.phase>
```

Comment:
The verify phase is defined as "run any checks to verify the package is valid and meets the quality criteria", whereas prepare-package is defined as performing ant operation necessary to prepare a package for actual packaging. 
checkstyle I would argue should remain part of the verify phase since it is defining a quality criteria that needs to be met. Is there a good reason for changing it???

Reply:
With the phase set to "verify" the build fails with checkstyle errors, such as:
`[ERROR] ~/Documents/work/palisade/Palisade-common/mvn_dependency_tree.txt:1: Line does not match expected header line of '^(/\*|#)$'. [RegexpHeader]`
These checkstyle errors even occur on the files within the "target" directory which should not be checked. Having it in the "prepare-package" phase still allows checkstyle to check the files and report any errors but does not report any issues on the files within the "target" directory.